examples/local/blueprint.yaml

```diff
@@ -2,6 +2,7 @@ tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
   - http://www.getcloudify.org/spec/cloudify/4.5/types.yaml
+  # For use with a manager use: plugin:cloudify-openstacksdk-plugin
```

Comment:
This is a "Local" blueprint - so no need really for this ... but whatever.

Reply:
Agree but this is just in case anyone want to use it on manager